app/src/main/AndroidManifest.xml

```diff
@@ -23,80 +24,79 @@
+            android:excludeFromRecents="true"
+            android:finishOnTaskLaunch="true"
+            android:launchMode="singleInstance"
+            android:theme="@android:style/Theme.Dialog" />
         <activity android:name=".auth.LoginActivity">
-            <intent-filter>
-                <category android:name="android.intent.category.LAUNCHER" />
-                <action android:name="android.intent.action.MAIN" />
-            </intent-filter>
-        </activity>
 
+        </activity>
         <activity android:name=".WelcomeActivity" />
-
         <activity
             android:name=".upload.ShareActivity"
             android:icon="@drawable/ic_launcher"
             android:label="@string/app_name">
             <intent-filter>
                 <action android:name="android.intent.action.SEND" />
+
                 <category android:name="android.intent.category.DEFAULT" />
+
                 <data android:mimeType="image/*" />
                 <data android:mimeType="audio/ogg" />
             </intent-filter>
         </activity>
-
         <activity
             android:name=".upload.MultipleShareActivity"
             android:icon="@drawable/ic_launcher"
             android:label="@string/app_name">
             <intent-filter>
                 <action android:name="android.intent.action.SEND_MULTIPLE" />
+
                 <category android:name="android.intent.category.DEFAULT" />
+
                 <data android:mimeType="image/*" />
                 <data android:mimeType="audio/ogg" />
             </intent-filter>
         </activity>
-
         <activity
             android:name=".contributions.ContributionsActivity"
             android:icon="@drawable/ic_launcher"
             android:label="@string/app_name" />
-
         <activity
             android:name=".settings.SettingsActivity"
             android:label="@string/title_activity_settings" />
-
         <activity
             android:name=".AboutActivity"
             android:label="@string/title_activity_about"
             android:parentActivityName=".contributions.ContributionsActivity" />
-
         <activity
             android:name=".auth.SignupActivity"
             android:label="@string/title_activity_signup" />
-
         <activity
             android:name=".nearby.NearbyActivity"
             android:label="@string/title_activity_nearby"
             android:parentActivityName=".contributions.ContributionsActivity" />
-
         <activity
             android:name=".notification.NotificationActivity"
             android:label="@string/navigation_item_notification" />
-
         <activity
             android:name=".featured.FeaturedImagesActivity"
             android:label="@string/title_activity_featured_images" />
+        <activity
+            android:name=".Achievements"
+            android:label="@string/Achievements">
+            <intent-filter>
+                <category android:name="android.intent.category.LAUNCHER" />
```

Comment:
Do we really want a launcher for this?

Reply:
It is basically for testing purposes. As soon as I change I will wire it to navigation drawer I will change back to its original entry point.